src/utils/format-numbers.js

```diff
@@ -161,6 +161,31 @@ export function formatSeconds(
   );
 }
 
+export function formatTimestamp(
+  time: Milliseconds,
+  significantDigits: number = 5,
+  maxFractionalDigits: number = 3
+) {
+  // Format in the closest base (seconds, milliseconds, microseconds, or nanoseconds),
+  // to avoid cases where times are displayed with too many leading zeroes to be useful.
+  if (time > 1000) {
```

Comment:
Just wondering why you used `>` instead of `>=` here :-) Do you prefer `1000ms` to `1s`? I would thing the latter is better, what do you think?

Reply:
Woops, that's a typo.